### PR TITLE
Update npm deps and test-bemtree bundle

### LINKED ISSUE #709: Update npm dependencies

### PR DESCRIPTION
**Pull request by tadatuta**
Closes #709 

// cc @narqo

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,12 +15,12 @@
   "license": "MIT",
   "dependencies": {
     "ym": "0.1.0",
-    "bem-xjst": "0.8.3",
-    "bemhtml-compat": "~0.0.11",
-    "bem-environ": "~1.2.0"
+    "bem-xjst": "0.9.0",
```
Comment (narqo) [inline package.json:18]: Does this update suite semver? Is it backward compatible*?

*just asking

**Comment by narqo** [comment]
Looks fine, but why Travis build failed?

File: package.json (hunk 1 above)
Comment (tadatuta) [inline package.json:18]: as it's always with `bem-xjst`, I can't say for sure, but as far as I understand the idea of the update is:
1. Make `this._mode` the first grouping in compiled result
2. Update minor version of `xjst` (actually there's `~` dependency anyway)

**Comment by tadatuta** [comment]
It was mocha timeout. Just restarted travis and it's fine now.

**Pull request merged**